src/brod.erl

```diff
@@ -1347,6 +1368,55 @@ fetch_committed_offsets(Client, GroupId) ->
 main(X) -> brod_cli:main(X).
 -endif.
 
+%% @doc starts a new transaction, TxId will be the id of the transaction
```

Comment:
Nit: coding style: the rest of the module has comments written in imperative mood, e.g. "Start a new transaction" rather than "starts a new transaction". Also we extract edocs from the code (see: https://hexdocs.pm/brod/brod.html#functions), so it makes sense to write the comments with users in mind and follow the markup (e.g. wrap variable names in `')